qcodes/instrument_drivers/Keysight/Infiniium.py

```diff
@@ -527,6 +527,12 @@ def __init__(self, name, address, timeout=20, **kwargs):
                             unit='s',
                             get_parser=float
                             )
+        
+        self.add_parameter('data_format',
+                           set_cmd='SAV:WAV:FORM {}',
+                           vals=Enum('CSV', 'ASC', 'BIN'),
```

Comment:
i'd strongly recommend to use a `val_mapping` here in order to hide the VISA values, for example:
```python
val_mapping={'csv': 'CSV', 'asc': 'ASC', 'binary': 'BIN'}
```
(would be nice to have something meaningful for the "ASC" case...)

Reply:
I'm new to github, so don't know how to edit my submission. For the record though, "asc" is short for "ASCIIXY", but I don't have any real understanding of what an ASCIIXY file is.

Reply:
I how now added val_mapping, but of course SCPI doesn't care about case, and accepts both the full name of the file type and the short-hand, so to my mind it would be preferable to have the validator accept "ASC", "ASCIIXY", "BIN", "BINARY", "CSV" and all case variations (i.e. make the validator completely case-insensitive), which would remove the need for any value mapping.